src/runner.go

```diff
@@ -71,8 +72,8 @@ func verifyYamlFile(yamlData []byte) bool {
 // If signature is valid then extracts the bash script to temporary file,
 // sets env variables if present and then runs the script.
 // Return stdout of executed script or error message if the signature wasn't valid.
-func processSignedScript(yamlFileContet []byte) string {
-	signatureIsValid := verifyYamlFile(yamlFileContet)
+func processSignedScript(ctx context.Context, yamlFileContet []byte) string {
```

Comment:
While we are here, what do you think about renaming `yamlFileContet` to `incomingContent`? 

It has a typo in the word `Content`, and since `yamlFileContent` is already being used inside the function, we could rename to something like: 

 ```suggestion
func processSignedScript(ctx context.Context, incomingContent []byte) string {
```

Or any other word that fits the idea.

Reply:
great suggestion! I was struggling to name it because I didn't wanna call it script or playbook (since it is technically neither)

Reply:
renamed